python_modules/dagster/dagster/_core/definitions/auto_materialize_policy.py

```diff
@@ -83,22 +87,40 @@ def time_window_partition_scope(self) -> Optional[datetime.timedelta]:
 
     @public
     @staticmethod
-    def eager() -> "AutoMaterializePolicy":
+    def eager(max_materializations_per_minute: Optional[int] = 1) -> "AutoMaterializePolicy":
+        """Args:
+        max_materializations_per_minute (Optional[int]): The maximum number of auto-materializations
+            for this asset that may be initiated per minute. If this limit is exceeded, the partitions
+            which would have been materialized will be discarded, and will require manual
+            materialization in order to be updated. Defaults to 1.
+        """
         return AutoMaterializePolicy(
             on_missing=True,
             on_new_parent_data=True,
             for_freshness=True,
             time_window_partition_scope_minutes=datetime.timedelta.resolution.total_seconds() / 60,
+            max_materializations_per_minute=check.opt_int_param(
+                max_materializations_per_minute, "max_materializations_per_minute"
+            ),
         )
 
     @public
     @staticmethod
-    def lazy() -> "AutoMaterializePolicy":
+    def lazy(max_materializations_per_minute: Optional[int] = 1) -> "AutoMaterializePolicy":
+        """Args:
```

Comment:
I think this will look nicer if you make the first line "Constructs a lazy AutoMaterializePolicy"